manageiq-operator/pkg/helpers/miq-components/httpd_conf.go

```diff
@@ -31,11 +31,11 @@ Options SymLinksIfOwnerMatch
 
   ProxyPreserveHost on
 
-  RewriteCond %{REQUEST_URI}     ^/ws        [NC]
```

Comment:
Is `/ws/notifications` the only thing that is requested under `/ws`?

Reply:
No, but I have a separate issue to deal with that https://github.com/ManageIQ/manageiq-pods/issues/536

Reply:
Also, this is based on https://github.com/ManageIQ/manageiq-appliance/blob/a1ba7b9b1bb91de046d93b6cd834e05a27f4e54f/COPY/etc/httpd/conf.d/manageiq-redirects-ui#L11-L12 I'm not sure why the regular `ProxyPass` didn't work for this, but either way only `/ws/notifications` goes to the UI